octomap_server/launch/octomap_tracking_server.launch

```diff
@@ -1,6 +1,10 @@
 <launch>
+
+  <arg name="path" default=""/>
+  <arg name="changeIdFrame" default="/talker/changes"/>
```

Comment:
I think it is actually easier to set a value 'on the fly' while calling a roslaunch.